packages/amplify-cli-core/src/utils/gql-transformer-version.ts

```diff
@@ -1,37 +0,0 @@
-import { $TSContext } from '../index';
```

Comment:
Can this file be deleted?

Reply:
Yep, the logic was a dup of that in the provider, probably moved here to make it easier to access. Now that you can call the ApiCategoryFacade.getTransformerVersion, the methods exposed here can go away.